services/QuillLMS/db/migrate/20210614205654_create_app_settings.rb

```diff
@@ -0,0 +1,14 @@
+class CreateAppSettings < ActiveRecord::Migration
+  def change
+    create_table :app_settings do |t|
+      t.string :name, null: false
+      t.string :user_ids_allow_list, null: false, default: [], array: true
```

Comment:
Since these are IDs, should we be using an integer for this column?  It probably doesn't really matter.